### PR TITLE
Upgrade node modules

### PR DESCRIPTION
**Pull request by psoleckimoj**
- New linting rules introduced
- Fix linting problems

### CODE REVIEW

File: server/auth/middleware.js [hunk 1]
```diff
@@ -97,7 +95,6 @@ const createSignOutMiddleware = ({ logger, analyticsService }) => {
     });
     res.redirect(req.query.returnUrl || '/');
```
Comment (LucasCairns) [inline server/auth/middleware.js:96]: Regarding the security advisory here, we could put a check in that the URL is relative, or ensure it is relative. So either;
`if (url.indexOf('://') > 0 || url.indexOf('//') === 0)`
or
`Url.parse(url).path()`

Comment (psoleckimoj) [reply server/auth/middleware.js:96]: No security issues on this PR 🤷‍♂️ 

it can get done separately.

Comment (psoleckimoj) [reply server/auth/middleware.js:96]: Well, there's a warning

Comment (psoleckimoj) [reply server/auth/middleware.js:96]: Has someone reenabled this or something as it's now picking up stuff that's been in forever as security issues? Which is good but bad that they've not been picked up before.

Comment (LucasCairns) [reply server/auth/middleware.js:96]: Ah okay, it showed up for me as a warning. I'll add it as a card to the Tech Debt column 👍

Comment (psoleckimoj) [reply server/auth/middleware.js:96]: @LucasCairns I'll card up this issue. It's not new and it's not failing the build. If you can check the PR for the change it's supposed to contain please.

**Review by LucasCairns — APPROVED** [review]
LGTM 👍

**Pull request merged**